Resource page updated

YamanSammani opened the pull request:
Kindly find attached a clean version of ResourcePage PR for your kind review. 


<img width="485" alt="Screenshot 2020-11-20 at 11 26 13" src="https://user-images.githubusercontent.com/67820326/99795315-7911ee00-2b23-11eb-8f9b-b93272345adf.png">

YamanSammani committed:
creat clean Resource Page, merge with dev

YamanSammani committed:
update test

tmansuroglu commented on src/locales/tr.json:
```diff
@@ -232,5 +232,9 @@
     "whyWeDoItText": "İnsanların doğayı ve özelliklerini korumak gibi ahlaki bir yükümlülüğü vardır, çünkü çevreyi korumak insanlığı toksik maddelerden ve gıda kalitemizi korur. MENA'daki yeni neslin, yerel ve küresel ölçekte insani ve sosyal konulara inanmak ve fayda sağlamak için hareket etmek için doğru araçlarla desteklenmesi gerektiğine inanıyoruz. Doğru araçlar verildiğinde, bir nesil aktiv ...
+  },
+  "resourcesPage": {
+    "title": "Kaynaklar Merkezi",
```
If this is an online library (I had no idea :D) you can call this page `Kütüphane` . `Kaynaklar Merkezi` sounds weird no matter what.

tmansuroglu reviewed (commented):
This PR changes unrelated snapshots and firebase config. Also there are some basic errors like leftover console log. I will review once you clear the code.

louisrli commented on src/components/Navbar/__snapshots__/index.test.js.snap:
```diff
@@ -220,7 +219,6 @@ exports[`Navbar Component matches the tree 1`] = `
         >
           <svg
             aria-hidden="true"
-            className=""
```
why is there a change in navbar here

YamanSammani replied:
when I run test, it throws some errors, so I clicked "a" to update all tests. But why it changes in other components I have no idea. I had a meeting with Ammar this morning and he told me it should be ok.  do you have any recommendation for this ?

louisrli commented on src/components/Resource/__snapshots__/index.test.js.snap:
```diff
@@ -0,0 +1,7 @@
+
+exports[`Resource renders correctly 1`] = `
+<h2>
+  Loading
```
this snapshot seems wrong

YamanSammani replied:
it was removed as you requested.

louisrli commented on src/components/Resource/index.jsx:
```diff
@@ -0,0 +1,51 @@
+    alert(`Couldn't get the download link`);
+  }
+};
+const Resource = ({ resource }) => {
```
newline

louisrli commented on src/components/Resource/index.test.js:
```diff
@@ -0,0 +1,8 @@
+import renderer from "react-test-renderer";
+import Resource from "./index.jsx";
+
+it("Resource renders correctly", () => {
```
just delete this test. there's not time but normally you'd mock the fetch

YamanSammani commented:
> This PR changes unrelated snapshots and firebase config. Also there are some basic errors like leftover console log. I will review once you clear the code.

@tmansuroglu , I had a meeting today with Ammar to fix an issue with the test and we updated the firebase config as I already added a part regarding authorization. So that's why there is a change in it. 
regarding the snapshots, I have no idea about it, I ask also Ammar and he said no need to worried about it as I didn't make these changes manually. I'm not an expert 😅

louisrli commented on src/containers/ResourcesPage/index.jsx:
```diff
@@ -0,0 +1,64 @@
+  },
+  {
+    id: 3,
+    title: "Who Causes Climate changes - TROCAIRE. ",
```
extra space

louisrli replied:
and theres a period here but not one at the end of line 21

louisrli commented on src/components/Resource/index.jsx:
```diff
@@ -0,0 +1,51 @@
+        <img
+          src={resource.pic}
+          alt={resource.title}
+          className=""
```
empty

YamanSammani replied:
I think it's added by mistake. since there is a className already.

YamanSammani committed:
updating feedback - remove test for Resource

YamanSammani commented:
@louisrli  
@tmansuroglu 
PTAL. Thanks.

louisrli commented:
make tests pass

YamanSammani committed:
Merge branch 'dev' of github.com:ReCoded-Org/capstone-istanbul-juniper into resourcePageUpdated

YamanSammani committed:
upate test

YamanSammani committed:
Merge branch 'dev' of github.com:ReCoded-Org/capstone-istanbul-juniper into resourcePageUpdated

louisrli commented:
merge conflicts

YamanSammani committed:
fix conflict

YamanSammani commented:
@louisrli  PTAL. Thanks.

YamanSammani commented:
@Ammar-64 , Please merge with dev.

Ammar-64 commented:
> @Ammar-64 , Please merge with dev. 

@ibrahimaljarad review is required

Ammar-64 committed:
Merge branch 'dev' into resourcePageUpdated
